doc/classes/ImmediateGeometry.xml

```diff
@@ -1,9 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="ImmediateGeometry" inherits="GeometryInstance" category="Core" version="3.0.alpha.custom_build">
 	<brief_description>
-		Node to draw simple geometry from code, ala OpenGL 1.x
+		Node to draw simple geometry from code, ala OpenGL 1.x.
 	</brief_description>
 	<description>
+		Node to draw simple geometry from code, ala OpenGL 1.x.
```

Comment:
"a la" , but I think there should be more detail here. Change the comma to a period and add something like:

    Uses a drawing mode similar to OpenGL 1.x.